python/tests/test_compare_conservation_laws_sbml.py

```diff
@@ -200,12 +201,11 @@ def test_adjoint_pre_and_post_equilibration(edata_fixture):
             assert np.isclose(rfa_cl['sllh'], raa_cl['sllh']).all()
             assert np.isclose(raa_cl['sllh'], rff_cl['sllh']).all()
 
-            # run simulations with different ways of preequilibration
-            # using a model with singular Jacobian
+    # --- compare fully adjoint approach to simulation with singular Jacobian ----
             raa = get_results(model, edata=edata, sensi_order=1,
                               sensi_meth=amici.SensitivityMethod.adjoint,
                               sensi_meth_preeq=amici.SensitivityMethod.adjoint,
                               reinitialize_states=reinit)
-                              
+
```

Comment:
what about combinations of pre and postequilibration?

Reply:
This is covered by this test case when running with edata_full from the fixture.

Reply:
If we're doing adjoint, we don't reach that point anyway but have returned before.